.yarn/patches/@metamask-assets-controllers-npm-34.0.0-ea790e90a1.patch

```diff
@@ -0,0 +1,123 @@
+diff --git a/dist/chunk-354SINOH.js b/dist/chunk-354SINOH.js
```

Comment:
Hey Pedro! I added a link to my core PR https://github.com/MetaMask/core/pull/4443 in the description;

I added the patch because i wanted to have one more approval on the core PR before merging. But i agree that ideally, upgrading assets-controllers would be better..